### ISSUE TITLE
Always prioritize higher quality vbr Opus audio over cvbr AAC

### ISSUE THREAD

**Issue by Ciancy28** [libre-tube/LibreTube#5145]
### Steps to reproduce

1. start watching a video
2. check audio codec in stats for nerds

### Expected behavior

where available, 128 kbps VBR Opus audio should always take priority (just like on the official YouTube app and website), as it's higher quality, and often uses less bandwidth, than the legacy 128 kbps CVBR AAC audio

### Actual behavior

Sometimes, the CVBR AAC audio is played instead (I think what's happening is that the audio track is chosen based on bitrate, which doesn't make sense, as the Opus VBR alghoritm aims at a constant quality, a mostly silent ~Opus 100 kbps track isn't lower quality than a 128 kbps AAC one just because it's lower bitrate).   
You can easily observe this behaviour when playing videos with a completely silent audio track (such as this one: https://youtu.be/g4mHPeMGTJM), libretube always streams the 128 kbps AAC track, which wastes a lot of bandwidth compared to the VBR opus one that's less than 4 kbps.   
Another advantage of the opus audio stream is that it matches android's audio subsystem's native 48 khz sample rate (less resource usage)
TL:DR: libretube should mimic yt-dlp's "bestaudio" behaviour instead of choosing based on bitrate

### LibreTube version

0.19.0

### Android version

Android 13

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
We let the `ExoPlayer` library handle the precedence of audio tracks (mime types, quality, ...).

I feel like this issue should be opened upstream at the library as we only have a very minimal impact on the chosen track ourself, as the `ExoPlayer` library includes the magic for fitting the audio track to the available network capacities. We can ask the player to prefer certain sample mime types, but that's not useful in this case anyway.

**Comment by Ciancy28** [comment]
> We can ask the player to prefer certain sample mime types, but that's not useful in this case anyway.

Wouldn't asking the player to prefer the webm mime type for the audio solve the issue? AAC is served in mp4 (m4a) container instead.    
Newpipe for example, lets the user choose between m4a (aac) and webm (opus) audio in the settings (I think they also use exoplayer), and if webm is chosen (which isn't the default but should be imo), the m4a stream is only served for those (very rare) videos that don't have opus webm audio available.
